test/test_pytree.py

```diff
@@ -274,18 +275,28 @@ def test_broadcast_to_and_flatten(self):
             result = _broadcast_to_and_flatten(pytree, to_spec)
             self.assertEqual(result, expected, msg=str([pytree, to_spec, expected]))
 
-    @parametrize("spec, str_spec", [
-        (TreeSpec(list, None, []), "L()"),
-        (TreeSpec(tuple, None, []), "T()"),
-        (TreeSpec(dict, [], []), "D()"),
-        (TreeSpec(list, None, [LeafSpec()]), "L(*)"),
-        (TreeSpec(list, None, [LeafSpec(), LeafSpec()]), "L(*,*)"),
-        (TreeSpec(tuple, None, [LeafSpec(), LeafSpec(), LeafSpec()]), "T(*,*,*)"),
-        (
-            TreeSpec(dict, ['a', 'b', 'c'], [LeafSpec(), LeafSpec(), LeafSpec()]),
-            "D(a:*,b:*,c:*)"
-        ),
-        (TreeSpec(list, None, [
+    @parametrize("spec", [
+        TreeSpec(list, None, []),
+        TreeSpec(tuple, None, []),
```

Comment:
Needs namedtuple for coverage, we have a custom rule for it

Reply:
there's a `test_pytree_serialize_namedtuple`